code/_onclick/hud/alert.dm

```diff
@@ -310,7 +310,7 @@ so as to remain in compliance with the most up-to-date laws."
 
 /obj/screen/alert/restrained/buckled
 	name = "Buckled"
-	desc = "You've been buckled to something and can't move. Click the alert to unbuckle unless you're handcuffed."
+	desc = "You've been buckled to something. Click the alert to unbuckle unless you're handcuffed."
```

Comment:
The problem is you aren't always unable to move, see https://github.com/tgstation/-tg-station/issues/12893